packages/11ty/_plugins/transforms/outputs/epub/manifest.js

```diff
@@ -85,10 +85,16 @@ module.exports = (eleventyConfig) => {
    * Publication title, subtitle, and reading line
    */
   const pubTitle = () => {
-    if (subtitle && readingLine) {
-      return `${title}: ${subtitle} ${readingLine}`
-    } else if (subtitle) {
-      return `${title}: ${subtitle}`
+    const joiner = title.endsWith('?') || title.endsWith('!') ? '' : ':';
```

Comment:
More robustly, if the title does not end with punctuation, use a separator.
```suggestion
    const separator = title.match(/[.,:!?]$/ '' : ':';
```

Reply:
An alternate functional approach
```
function endsWithPunctuation (string) {
  const punctuation = '.,:!?';
  return punctuation.some((character) => string.endsWith(character));
}
```